packages/cli/src/smoke-tests/smoke-tests.js

```diff
@@ -204,6 +204,34 @@ describe('smoke tests - setup will take some time', function () {
     result.should.be.Array();
   });
 
+  describe('zapier register should validate enum fields that are passed in as flags', function () {
+    it('zapier register should throw error for invalid role', () => {
+      (() => {
+        runCommand(context.cliBin, ['register', '--role', 'invalidRole']);
+      }).should.throw(/invalidRole is not a valid value for role/);
+    });
+
+    it('zapier register should throw error for invalid audience', () => {
+      (() => {
+        runCommand(context.cliBin, [
+          'register',
+          '--audience',
+          'invalidAudience',
+        ]);
+      }).should.throw(/invalidAudience is not a valid value for audience/);
+    });
+
+    it('zapier register should throw error for invalid category', () => {
+      (() => {
+        runCommand(context.cliBin, [
+          'register',
+          '--category',
+          'invalidCategory',
+        ]);
+      }).should.throw(/invalidCategory is not a valid value for category/);
+    });
+  });
```

Comment:
It would be nice to have for a valid value for each flag too.

Reply:
@kola-er I mainly wanted to test the enum fields since I wound up implementing [manual flag validation](https://github.com/zapier/zapier-platform/blob/4af40fd0b3b7958bc96799095d6fa247b54d07ec/packages/cli/src/oclif/commands/register.js#L26) for them. Is there anything in particular you'd like us to be testing for the other fields?

Reply:
Just to test for when the validation succeeds as we have (all) for when the validation fails.

Reply:
@kola-er Ohhhh I see, I misunderstood your first comment. I will add these tests first thing tomorrow!

Reply:
@kola-er Added in c9ebfe474189855794f098b60c331501ef0c2467